readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://dianaavem.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://dianaavem.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
launch command `npm run deploy` and add this links to pr description

Reply:
Don't forget to add links to the description